src/components/TokenField/TokenSelector/TokenOptions.tsx

```diff
@@ -140,18 +140,18 @@ function TokenCombobox({
       by="token_id"
       value={token}
       virtual={{ options: searchResult }}
-      onChange={async (token) => {
-        if (!token) return;
-        if (isApToken(token)) return onChange(token);
+      onChange={async (tkn) => {
```

Comment:
so will not clash with outside `token`
```suggestion
      onChange={async (token) => {
```